ocd_backend/extractors/ibabs.py

```diff
@@ -62,19 +61,53 @@ def run(self):
 
         if meeting_types.Meetingtypes:
             total_count = 0
-            for mt in meeting_types.Meetingtypes['iBabsMeetingtype']:
+            for mt in meeting_types.Meetingtypes[0]:
```

Comment:
[Magic numbers](https://en.wikipedia.org/wiki/Magic_number_%28programming%29#Unnamed_numerical_constants) usually are a bad idea and lead to hard to read code, they're in multiple places (e.g. `users.Users[0]`), what does the first position mean?

Reply:
`iterable[x]` is the way in Python to access an item in a list or dictionary by position. The output from the SOAP client is a dictionary with one element which is the actual output.

I didn't really change anything about this file, I just rolled it back to the way it was a few commits ago before we tried a different SOAP client.